tests/run/unicodemethods.pyx

```diff
@@ -156,23 +156,23 @@ def splitlines_keep_bint(unicode s):
     <BLANKLINE>
     sdflk as sa
     <BLANKLINE>
-    sadas asdas fsdf 
+    sadas asdas fsdf
     >>> print_all( multiline_text.splitlines(False) )
     ab jd
     sdflk as sa
-    sadas asdas fsdf 
+    sadas asdas fsdf
     >>> len(splitlines_keep_bint(multiline_text))
     7
     >>> print_all( splitlines_keep_bint(multiline_text) )
     ab jd
     <BLANKLINE>
     sdflk as sa
     <BLANKLINE>
-    sadas asdas fsdf 
+    sadas asdas fsdf
     --
     ab jd
     sdflk as sa
-    sadas asdas fsdf 
```

Comment:
It looks like my editor "helpfully" corrects the trailing whitespace in this file. I'll undo it